composer.json

```diff
@@ -28,7 +28,7 @@
         "doctrine/doctrine-fixtures-bundle":    "~2.2",
         "doctrine/doctrine-migrations-bundle":  "~1.0",
         "doctrine/migrations":                  "~1.0@dev",
-        "doctrine/orm":                         "~2.4",
+        "doctrine/orm":                         "~2.4.0",
```

Comment:
While using Doctrine 2.5 I run into some errors:
- It changed some method used in many specs, so they should be changed too
- It made some Behat suite not passing (I think it was country management in backend)